tests/test_service.py

```diff
@@ -578,7 +578,7 @@ def test_get_recommendation_relationship_type(self):
         """ Get recommendation relationship type for two products Tests"""
         valid_recommendation = self._create_recommendations(count=1)[0][0]
         resp = self.app.get(
-            "/recommendations/relationship",
+            BASE_URL + "/relationship",
             query_string=dict(
                 product1=valid_recommendation.product_id,
```

Comment:
you should rename all these "product1", "product2" in the unit tests to "product-id" and "related-product-id"